package.json

```diff
@@ -5,22 +5,31 @@
     "typings": "./dist/typings/index.d.ts",
     "main": "dist/commonjs/index.js",
     "types": "dist/typings/index.ts",
+    "bin": {
+        "fusebox": "bin/fsbx"
+    },
     "directories": {
+        "bin": "bin",
         "test": "test",
         "modules": "modules",
         "lib": "dist/commonjs",
-        "typings": "dist/typings"
+        "typings": "dist/typings",
+        "docs": "docs"
     },
     "files": [
+        "bin",
         "src",
         "dist",
         "modules",
         "test",
         "docs"
     ],
     "scripts": {
+        "test-cli": "fusebox",
         "help": "node bin/fsbx --help",
-        "test": "TRAVIS=true ./node_modules/mocha/bin/mocha && node ./test.js",
+        "lint-fix": "eslint src/*.ts src/**/*.ts src/**/**/*.ts test/*.js test/**/*.js --fix",
```

Comment:
get your point.I've encountered this problem too. The eslint default parser may not work well with typescript. if you don't want to use a experimental parser, I think a better way is to use [tslint](https://github.com/palantir/tslint).